src/org/pahappa/systems/exceptions/SavingFailedException.java

```diff
@@ -1,9 +1,9 @@
 package org.pahappa.systems.exceptions;
 public class SavingFailedException extends Exception {
 
-	/**
-	 * 
-	 */
+	public SavingFailedException(String message){
+		super("Incident not saved because" + message);
+	}
 	private static final long serialVersionUID = 1L;
```

Comment:
usually better if the constant is before the method definition

Reply:
@MutebiAlexKato kindly attend to this comment